### ISSUE TITLE
Cross-compile (target=x86_64-pc-windows-gnu) from linux fails

### ISSUE THREAD

**Issue by jmetz** [norepimorphism/libui-ng-sys#3]
Technically this error occurs in boing, but is caused by this package. 

Get error about missing winres file when attempting to target windows from linux: 

```
  package.metadata.winres does not exist

  Error: IncludeWinres(Os { code: 2, kind: NotFound, message: "No such file or directory" })
```

**Comment by norepimorphism** [comment]
Thanks for including the target triple. Your target is `x86_64-pc-windows-gnu`, the definition of which is found here:
https://github.com/rust-lang/rust/blob/742d3f02c243964e5b868d90afd60c2907be5853/compiler/rustc_target/src/spec/windows_gnu_base.rs#L73-L105

`rustc` sees that your target implies a `"windows"` family, so it defines `cfg(windows)` here:
https://github.com/rust-lang/rust/blob/ce61d4044d7621fd190a2ddd1f51fa78e2941518/compiler/rustc_session/src/config.rs#L959-L966

When `winres` then queries the target family, it selects a default `windres` path of `windres.exe`:
https://github.com/mxre/winres/blob/1807bec3552cd2f0d0544420584d6d78be5e3636/lib.rs#L218-L221

Of course, your host is Linux, so `winres` cannot locate the file. I imagine this is the cause of the `"No such file or directory"` error that you observed above.

While I think this issue is best resolved in `winres` itself so that others cross-compiling from Linux to Windows won't experience the same error, unfortunately, a PR that appears to fix the issue (mxre/winres#36) has been stagnating for over a year, and the last commit to the `winres` repository occurred 14 months ago.

As `winres` appears inactive, I will issue a fix on the `libui-ng-sys` side of things. Thanks for reporting the issue!

**Comment by norepimorphism** [comment]
It seems that, in the time since I made that reply, more activity has occurred on the aforementioned `winres` issue thread, resulting in the release of a [`winresource`](https://crates.io/crates/winresource) crate that ostensibly resolves the original issue.

However, rather than swapping `winres` for this new crate, I am now considering removing Windows manifest functionality from `libui-ng-sys` altogether. (I might do it in the next few hours, actually, while I still have some energy.) Once I do so, this issue should be resolved.

`libui-ng-sys`' manifest was merely a half-baked fix for a lack of DPI awareness in *libui-ng* that could be overridden anyway by a secondary manifest. I recommend users of `libui-ng-sys` that have been relying on the built-in manifest for DPI awarenness to apply your own manifest because that is what you should be doing anyway. (Note: I may be one such user myself.)
